### PR TITLE
added github issue/pr templates and contributing guidelines

### PR DESCRIPTION
**Pull request by ghuntley**

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+**What is the new behavior (if this is a feature change)?**
+
+
+
+**Does this PR introduce a breaking change?**
+
+
+
+**Please check if the PR fulfills these requirements**
+- [ ] The commit follows our guidelines: https://github.com/paulcbetts/refit/blob/master/CONTRIBUTING.md
```
Comment (bennor) [inline .github/PULL_REQUEST_TEMPLATE.md]: This doesn't exist yet. Are you going to add it?

Comment (onovotny) [inline .github/PULL_REQUEST_TEMPLATE.md]: would be great to get that include with this PR to make it complete

**Comment by ghuntley** [comment]
Ready for review/merge.

**Comment by worldbeater** [comment]
Hi there from 2019! :smile: Refit now uses organization-wide issue/pr templates from [reactiveui .github](https://github.com/reactiveui/.github/tree/master/.github) repository. So probably worth leaving this PR only for `CONTRIBUTING.md` and then merging.

**Comment by clairernovotny** [comment]
Closing due to age

**Pull request closed without merging**